packages/config/config/devices/0x0344/he-rs01.json

```diff
@@ -0,0 +1,862 @@
+				{
+					"label": "Controlled by external input S1",
+					"value": 1
+				},
+				{
+					"label": "Controlled by external input S2",
+					"value": 2
+				},
+				{
+					"label": "Controlled by external input S3",
+					"value": 3
+				},
+				{
+					"label": "Controlled by external input S4",
+					"value": 4
+				},
+				{
+					"label": "Controlled by external input S5",
+					"value": 5
+				}
+			]
+		},
+		"71": {
+			"label": "Relay 1 Timer Mode Duration",
+			"description": "This function can be used to open/close garage doors, blinds, curtains, etc. or to turn attached devices like door lock or security OFF for a short time. If the parameter value is set to ―0‖ it will operate as a short contact (about 0.5sec).",
+			"unit": "seconds",
+			"valueSize": 3,
+			"minValue": 0,
+			"maxValue": 43200,
+			"defaultValue": 0
+		},
+		"72": {
+			"label": "Relay 2 Timer Mode Duration",
+			"description": "This function can be used to open/close garage doors, blinds, curtains, etc. or to turn attached devices like door lock or security OFF for a short time. If the parameter value is set to ―0‖ it will operate as a short contact (about 0.5sec).",
+			"unit": "seconds",
+			"valueSize": 3,
+			"minValue": 0,
+			"maxValue": 43200,
+			"defaultValue": 0
+		},
+		"73": {
+			"label": "Relay 3 Timer Mode Duration",
+			"description": "This function can be used to open/close garage doors, blinds, curtains, etc. or to turn attached devices like door lock or security OFF for a short time. If the parameter value is set to ―0‖ it will operate as a short contact (about 0.5sec).",
+			"unit": "seconds",
+			"valueSize": 3,
+			"minValue": 0,
+			"maxValue": 43200,
+			"defaultValue": 0
+		},
+		"74": {
+			"label": "Relay 4 Timer Mode Duration",
+			"description": "This function can be used to open/close garage doors, blinds, curtains, etc. or to turn attached devices like door lock or security OFF for a short time. If the parameter value is set to ―0‖ it will operate as a short contact (about 0.5sec).",
+			"unit": "seconds",
+			"valueSize": 3,
+			"minValue": 0,
+			"maxValue": 43200,
+			"defaultValue": 0
+		},
+		"75": {
+			"label": "Relay 5 Timer Mode Duration",
+			"description": "This function can be used to open/close garage doors, blinds, curtains, etc. or to turn attached devices like door lock or security OFF for a short time. If the parameter value is set to ―0‖ it will operate as a short contact (about 0.5sec).",
+			"unit": "seconds",
+			"valueSize": 3,
+			"minValue": 0,
+			"maxValue": 43200,
+			"defaultValue": 0
+		},
+		"101": {
+			"label": "Relay Inverse Mode, Inching - Group 1",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 51,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Function is disabled",
+					"value": 0
+				},
+				{
+					"label": "Relay 1 and Relay 2",
+					"value": 12
+				},
+				{
+					"label": "Relay 2 and Relay 3",
+					"value": 23
+				}
+			]
```

Comment:
These values don't seem to make sense. Why is the max 51 when you can only select 0, 12 and 23?

Reply:
Fixed